src/System.Net.Http/tests/FunctionalTests/PostScenarioTest.cs

```diff
@@ -18,7 +18,7 @@ public abstract class PostScenarioTest : HttpClientTestBase
     {
         private const string ExpectedContent = "Test contest";
         private const string UserName = "user1";
-        private const string Password = "password1";
+        private const string UserName = "PLACEHOLDER";
```

Comment:
It seems that you changed `Password` to `UserName` by mistake?

Reply:
thanks for the catch!